File: MtxVec.Delphi/IntroPolynoms.pas
unit IntroPolynoms;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  Basic3, StdCtrls, ComCtrls;

type
  TIntroPoly = class(TBasicForm3)
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;
                                                               
var
  IntroPoly: TIntroPoly;

implementation

{$IFDEF CLR}
{$R *.NFM}
{$ELSE}
{$R *.DFM}
{$ENDIF}

procedure TIntroPoly.FormCreate(Sender: TObject);
begin
  inherited;
  with RichEdit1.Lines, RichEdit1 do
  begin
    Clear;
    SelAttributes.Style := [fsBold];
    SelAttributes.Color := RGB(51,104,196);
    SelAttributes.Size := 11;
    Add('Robust polynomials handling methods:');
    Add('');
    Paragraph.FirstIndent := 10;
    Paragraph.Numbering := nsBullet;
    Add('Polynomial evaluation, root finding');
    Add('Piece-wise polynomials');
    Add('Fast cubic spline algorithm');
    Add('Linear or cubic interpolation');
    Add('Fit high order polynomials to data');
    Paragraph.FirstIndent := 0;
    Paragraph.Numbering := nsNone;
  end;
end;

initialization
   RegisterClass(TIntroPoly);

end.

File: MtxVec.CPP/IntroPolynoms.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IntroPolynoms.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Basic3"
#pragma resource "*.dfm"
TIntroPoly *IntroPoly;
//---------------------------------------------------------------------------
__fastcall TIntroPoly::TIntroPoly(TComponent* Owner)
  : TBasicForm3(Owner)
{

//  TBasicForm3::FormCreate(Sender); // inherited

  RichEdit1->SelAttributes->Style = RichEdit1->SelAttributes->Style << TFontStyle::fsBold;
  RichEdit1->SelAttributes->Color = (TColor) RGB(51,104,196);
  RichEdit1->SelAttributes->Size = 11;
  RichEdit1->Lines->Add("Robust polynomials handling methods:");
  RichEdit1->Lines->Add("");
  RichEdit1->Paragraph->FirstIndent = 10;
  RichEdit1->Paragraph->Numbering = nsBullet;
  RichEdit1->Lines->Add("Polynomial evaluation, root finding");
  RichEdit1->Lines->Add("Piece-wise polynomials");
  RichEdit1->Lines->Add("Fast cubic spline algorithm");
  RichEdit1->Lines->Add("Linear or cubic interpolation");
  RichEdit1->Lines->Add("Fit high order polynomials to data");
  RichEdit1->Paragraph->FirstIndent = 0;
  RichEdit1->Paragraph->Numbering = nsNone;
}
//---------------------------------------------------------------------------

 